risc0/zkvm/sdk/rust/build/src/lib.rs

```diff
@@ -404,12 +404,18 @@ fn build_guest_package<P>(
 /// [`embed_methods_with_options`].
 pub struct GuestOptions {
     /// The number of po2 entries to generate in the MethodID.
```

Comment:
Perhaps just:

/// The number of <s>po2</s> entries <s>to generate</s> in the MethodID.